server/controllers/conversation.js

```diff
@@ -0,0 +1,111 @@
+const { validationResult } = require("express-validator");
+const User = require("../models/User");
+const Conversation = require("../models/Conversation");
+const Message = require("../models/Message");
+
+// @route POST /start
+// @desc start a new conversation with the logged in user and another valid user
+// @access Private
+exports.startConversation = async (req, res) => {
+  try {
```

Comment:
Note: We have `express-async-handler` installed which handles promise rejections (we don't need a try/catch block when we use the asyncHandler)

ref: https://www.npmjs.com/package/express-async-handler